src/server/ns_turn_server.c

```diff
@@ -1096,7 +1096,7 @@ static int handle_turn_allocate(turn_turnserver *server, ts_ur_super_session *ss
               *err_code = 442;
               *reason = (const uint8_t *)"UDP Transport is not allowed by the TURN Server configuration";
             } else if (ss->client_socket) {
-              SOCKET_TYPE cst = get_ioa_socket_type(ss->client_socket);
+              SOCKET_TYPE const cst = get_ioa_socket_type(ss->client_socket);
```

Comment:
Should find/replace all those `SOCKET_TYPE const` probably

Reply:
I believe this is addressed now.